mcp_hydrolix/mcp_env.py

```diff
@@ -177,9 +209,13 @@ def _validate_required_vars(self) -> None:
             ValueError: If any required environment variable is missing.
         """
         missing_vars = []
-        for var in ["HYDROLIX_HOST", "HYDROLIX_USER", "HYDROLIX_PASSWORD"]:
-            if var not in os.environ:
-                missing_vars.append(var)
+        if self.service_account:
+            required_vars = ["HYDROLIX_HOST", "HYDROLIX_TOKEN"]
+        else:
+            required_vars = ["HYDROLIX_HOST", "HYDROLIX_USER", "HYDROLIX_PASSWORD"]
+            for var in required_vars:
+                if var not in os.environ:
+                    missing_vars.append(var)
```

Comment:
```suggestion
        for var in required_vars:
            if var not in os.environ:
                missing_vars.append(var)
```
This should be checked regardless of which authentication method we're using (though which vars are required does change)